plugins/inputs/uwsgi/uwsgi_test.go

```diff
@@ -0,0 +1,125 @@
+package uwsgi_test
```

Comment:
also write a test with an http returning invalid JSON, to make sure that an error is returned there as well